### PR TITLE
E2E - (Mittente) Visualizzazione Dettagli di notifica con pagamento e documenti

### PR DESCRIPTION
**Pull request by azhansultan**
*Added new feature file and steps for tests PN-10393, PN-10390,PN-10387,PN-10386,PN-10385,PN-10384,PN-10383,PN-10382

### CODE REVIEW

File: src/main/java/it/pn/frontend/e2e/pages/mittente/PiattaformaNotifichePage.java [hunk 1]
```diff
@@ -978,4 +978,10 @@ public void checkMessaggioErroreConCodice(int code) {
         }
     }
 
+    public void clickVediTutti() {
+        By vediTutti = By.xpath("//button[@data-testid='show-all-attachments']");
+        getWebDriverWait(10).withMessage("Il bottone vedi tutti non cliccabile").until(ExpectedConditions.elementToBeClickable(vediTutti));
```
Comment (leleOFA) [inline src/main/java/it/pn/frontend/e2e/pages/mittente/PiattaformaNotifichePage.java]: insert ExpectedConditions.and(ExpExpectedConditions1, ExpectedConditions2) which check if is visible and clickable

File: src/test/resources/feature/2-mittente/4_visualizzazioneNotifiche/VisualizzaNotifica.feature [hunk 2]
```diff
@@ -0,0 +1,87 @@
+Feature: Mittente visualizza il dettaglio di una notifica con documenti allegati
+
+  @TestSuite
+  @TA_MittenteDettaglioNotificheConDocumenti
+  @mittente
+  @visualizzazioneNotificheMittente
+
+  @CheckNotificaConDocumentiAllegati
+  Scenario: PN-10393 - Mittente visualizza box allegati modelli F24
```
Comment (leleOFA) [inline src/test/resources/feature/2-mittente/4_visualizzazioneNotifiche/VisualizzaNotifica.feature]: change number with a small description of test doing

Comment (azhansultan) [reply src/test/resources/feature/2-mittente/4_visualizzazioneNotifiche/VisualizzaNotifica.feature]: which number?

File: src/test/resources/feature/2-mittente/4_visualizzazioneNotifiche/VisualizzaNotifica.feature [hunk 3]
```diff
@@ -16,7 +16,7 @@ Feature: Mittente visualizza il dettaglio di una notifica con documenti allegati
     And Logout da portale mittente
 
   @CheckNotificaConDocumentiAllegati
-  Scenario: PN-10390 - Mittente visualizza dettaglio notifica con attestazione opponibile a terzi notifica presa in carico
+  Scenario: [TA-FE VISUALLIZZAZIONE DETTAGLI DI NOTIFICA] - Mittente visualizza dettaglio notifica con attestazione opponibile a terzi notifica presa in carico
```
Comment (leleOFA) [inline src/test/resources/feature/2-mittente/4_visualizzazioneNotifiche/VisualizzaNotifica.feature]: this title should be different from other and unique, fix VISUALLIZZAZIONE in VISUALIZZAZIONE

File: src/main/java/it/pn/frontend/e2e/pages/mittente/PiattaformaNotifichePage.java [hunk 4]
```diff
@@ -980,7 +983,7 @@ public void checkMessaggioErroreConCodice(int code) {
 
     public void clickVediTutti() {
         By vediTutti = By.xpath("//button[@data-testid='show-all-attachments']");
-        getWebDriverWait(10).withMessage("Il bottone vedi tutti non cliccabile").until(ExpectedConditions.elementToBeClickable(vediTutti));
+        getWebDriverWait(10).withMessage("Il bottone vedi tutti non cliccabile").until(ExpectedConditions.and(visibilityOfElementLocated(vediTutti),elementToBeClickable(vediTutti)));
```
Comment (leleOFA) [inline src/main/java/it/pn/frontend/e2e/pages/mittente/PiattaformaNotifichePage.java]: this is ok, but restore other as before, delete this and decrease timeout

File: src/test/resources/feature/2-mittente/4_visualizzazioneNotifiche/VisualizzaNotifica.feature (hunk 2 above)
Comment (Gabriiii99) [reply src/test/resources/feature/2-mittente/4_visualizzazioneNotifiche/VisualizzaNotifica.feature]: The task number. Write something like this: 
[TA-FE DOWNLOAD FILE NOTIFICA CON PAGAMENTO] - Il destinatario accede ad una notifica con pagamento e scarica il modello F24

Comment (azhansultan) [reply src/test/resources/feature/2-mittente/4_visualizzazioneNotifiche/VisualizzaNotifica.feature]: Already done

**Review by leleOFA — APPROVED** [review]

**Pull request merged**